### PR TITLE
Add current_course_option_id to ApplicationChoice

### PR DESCRIPTION
**Pull request by mnacos**
This is the db migration for https://github.com/DFE-Digital/apply-for-teacher-training/pull/4499

We are renaming `current_option` to `current_course_option`, as suggested in the review.

### CODE REVIEW

**Review by despo — APPROVED** [review]
Looks good. Can we not merge until the other PR has also been approved - or is near approval - just in case there are any further changes.

**Comment by mnacos** [comment]
Discussed on Slack, will merge immediately as it's blocking renames in the other PR.

**Pull request merged**
